cmd/create.go

```diff
@@ -38,8 +38,23 @@ func init() {
 }
 
 var createCmd = &cobra.Command{
-	Use:        "create <name>",
-	Short:      "Create a new Function, including initialization of local files and deployment.",
+	Use:   "create <name>",
+	Short: "Create a new Function, including initialization of local files and deployment",
+	Long: `Create a new Function, including initialization of local files and deployment
+
+Creates a new Function project at 'path'. If 'path' does not exist, it is
```

Comment:
We are talking about 'path' in here, but it is not clear, where in the command the path should be specified.